### PR TITLE
Chore/reset database

### LINKED ISSUE #34: Update database with cropped images and remove word "tree" from tree names

### LINKED ISSUE #35: Update DB with more CO zipcodes

### LINKED ISSUE #36: Add updated attributes and individualize them on CSV and then DB

### PR DESCRIPTION
**Pull request by rachel-bock**
What does this PR do?
- Reset the data that was used to seed the database.

Where should your reviewer start?
- init.sql

What (if any) features are you implementing?
- N/A

What (if anything) did you refactor?
- N/A

Were there any issues that arose?
- There were some semi-colons where commas should have been and we added multiple growing zones for CO zip codes.

Is there anything that you need from your teammate?
- N/A

Any other comments, questions, or concerns?
- Resolves #36 
- Resolves #35 
- Resolves #34

### CODE REVIEW

File: old_ZipCodesTable.sql [hunk 1]
```diff
@@ -140,7 +140,7 @@ INSERT INTO zip_codes (id, sid, default_state, city, default_type, zip_code, har
 (5153, 7, 'CO', 'DENVER', 'UNIQUE', '80291', '5b'),
 (5154, 7, 'CO', 'DENVER', 'STANDARD', '80293', '5b'),
 (5155, 7, 'CO', 'DENVER', 'STANDARD', '80294', '5b'),
-(5156, 7, 'CO', 'DENVER', 'STANDARD', '80299', '5b')
+(5156, 7, 'CO', 'DENVER', 'STANDARD', '80299', '5b'),
```
Comment (BrandonAinsworth) [inline old_ZipCodesTable.sql:143]: Great attention to detail on display here, nice work!

File: Zips.sql [hunk 2]
```diff
@@ -0,0 +1,170 @@
+(5372, 7, 'CO', 'MOFFAT', 'STANDARD', '81143', 5),
```
Comment (BrandonAinsworth) [inline Zips.sql:1]: This helped resolve a merge conflict, we can find everything in init.sql

**Review by BrandonAinsworth — APPROVED** [review]

**Review by Baskanbetul — APPROVED** [review]
Great job for updating database 👩🏼‍💻

**Review by irmakerdem — APPROVED** [review]
Hooray more zipcodes! 💌 Nice job cleaning up the images! 🌠

**Pull request merged**